esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2214,8 +2214,38 @@ bool CFileSprayEx::onReplicate(IEspContext &context, IEspReplicate &req, IEspRep
     return true;
 }
 
+void CFileSprayEx::getDropZoneInfoByDestGroup(double clientVersion, const char* destGroup, const char* destFileIn, StringBuffer& destFileOut, StringBuffer& umask)
```

Comment:
Other places are referring to this as destGroup and it may be confusing to have the same field refered by 2 different names.  (The ESP services and eclwatch refer to this field as destGroup.)  Shall I rename this to destPlane still?